src/layouts/master.pug

```diff
@@ -5,10 +5,14 @@ html(xmlns='http://www.w3.org/1999/xhtml')
     block head
       meta(http-equiv='Content-Type' content='text/html; charset=utf-8')
       meta(name='viewport' content='width=device-width')
-
+      <!--[if mso]>
+      style(type='text/css')
+        != 'body, table, td {font-family: "Helvetica Neue", Helvetica, Arial, sans-serif !important;}'
+      <![endif]-->
+      
   body(leftmargin='0' marginheight='0' marginwidth='0' offset='0' topmargin='0')
     center
-      table(align='center' border='0' cellpadding='0' cellspacing='0' height='100%' id='bodyTable' width='100%')
+      table(align='center' border='0' cellpadding='0' cellspacing='0' height='100%' id='bodyTable' width='560')
```

Comment:
Do we want the email to be a fixed size? In browser this wouldn't be responsive below 560px. Also how do email email clients respond to this?

Taking into consideration larger font sizes set on devices for accessibility.